db/seeds.rb

```diff
@@ -5,3 +5,5 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+Admin.delete_all
+Admin.create!({email: "admin@receiptyourself.com", password: "receiptyourself", password_confirmation: "receiptyourself" })
```

Comment:
It works fine, Admin does not need to confirm email, only user has to do that. This credentials work fine after seed.